src/main/resources/static/js/common/view/EditView.js

```diff
@@ -1,11 +1,33 @@
 define([
     'backbone.marionette',
-    'backbone.stickit'
+    'backbone.stickit',
+    'backbone.validation'
 ], function () {
 
     'use strict';
 
     return Marionette.ItemView.extend({
+        initialize: function () {
+            Backbone.Validation.bind(this);
+
+            _.extend(Backbone.Validation.callbacks, {
+                valid  : function (view, attr, selector) {
+                    var $el    = view.$('#' + attr),
+                        $group = $el.closest('.form-group');
+
+                    $group.removeClass('has-error');
+                    $group.find('.help-block').html('').addClass('hidden');
+                },
+                invalid: function (view, attr, error, selector) {
+                    var $el    = view.$('#' + attr),
+                        $group = $el.closest('.form-group');
+
+                    $group.addClass('has-error');
+                    $group.find('.help-block').html(error).removeClass('hidden');
+                }
+            });
+        },
+
```

Comment:
Extend Validation callbacks in scope of application instead of one particular view.
Move this code to the ConfApp class